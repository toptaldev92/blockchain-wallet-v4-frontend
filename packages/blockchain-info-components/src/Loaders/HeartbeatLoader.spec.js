import { shallow } from 'enzyme'
import React from 'react'
import toJson from 'enzyme-to-json'

import HeartbeatLoader from './HeartbeatLoader'

describe('HeartbeatLoader component', () => {
  it('default renders correctly', () => {
    const component = shallow(
      <HeartbeatLoader height='10px' width='10px' color='blue900' />
    )
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
