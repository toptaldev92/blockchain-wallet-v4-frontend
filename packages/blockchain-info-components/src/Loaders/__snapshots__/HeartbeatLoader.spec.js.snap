// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeartbeatLoader component default renders correctly 1`] = `
<styled.div
  height="10px"
  width="10px"
>
  <styled.div
    color="blue900"
  />
  <Styled(styled.div)
    color="blue900"
  />
</styled.div>
`;
